backend/tests/ingestion/test_classification.py

```
@@ -0,0 +1,43 @@
+from litigation_api.ingestion.classification import DocumentClassifier, ClassificationProvider, classify_document
+
+class MockFallbackProvider(ClassificationProvider):
+    def classify(self, text: str, title: str | None = None, docket_title: str | None = None) -> str | None:
```

Ruff (0.15.11)

[warning] Unused method argument: `title`

(ARG002)